### PR TITLE
[multiple] Add licenses

### PR DESCRIPTION
**Pull request by bparrishMines**
Checks that all plugins correctly contains licenses.
This check enforces the following rules:
1. Every plugin must contain a LICENSE file in root directory.
2. Every LICENSE file must contain the copyright license with 'Flutter' or 'Chromium' as the author.
3. All source code files must contain a license header at the top.
4. The license header for every source code file must contain the same author as the LICENSE file in the plugin's root directory.

Submitting this despite ipa tests continue to timeout. This will be reverted if it doesn't pass the general test after submission.

### CODE REVIEW

File: packages/image_picker/LICENSE [hunk 1]
```diff
@@ -1,232 +1,27 @@
...
-A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
-OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
-SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
-LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
-DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
-THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
---------------------------------------------------------------------------------
-aFileChooser
```
Comment (cyanglaz) [inline packages/image_picker/LICENSE]: This is required as image picker is using the `aFileChooser` which uses apatch license.

Comment (bparrishMines) [reply packages/image_picker/LICENSE]: Yea it has to match the author name in the plugin `LICENSE` file. I added more info to the PR description.

Comment (bparrishMines) [reply packages/image_picker/LICENSE]: I also didn't mean to change this file. I had to update my script :)

**Comment by Hixie** [comment]
does this add a test to enforce it?

**Comment by bparrishMines** [comment]
@Hixie Yes, I created this PR that adds a command line tool that will enforce it and can automatically add them to your files: https://github.com/flutter/plugin_tools/pull/48

**Comment by amirh** [comment]
Please update the PR description to reflect what this PR is doing (add/update licenses)

**Pull request merged**